content/docs/styled-system/theme.mdx

```diff
@@ -365,21 +365,19 @@ Chakra provides a set of smooth corner radius values.
 ```js
 import { extendTheme } from '@chakra-ui/react'
 
-const borderRadius = {
-  radii: {
-    none: '0',
-    sm: '0.125rem',
-    base: '0.25rem',
-    md: '0.375rem',
-    lg: '0.5rem',
-    xl: '0.75rem',
-    '2xl': '1rem',
-    '3xl': '1.5rem',
-    full: '9999px',
-  },
-}
-
-const theme = extendTheme({ borderRadius, ...})
```

Comment:
To avoid confusion, we can keep the `borderRadius` name, but spread it instead inside of the `extendTheme`. That'd also be correct. We can also update the text above the code example to:
> Chakra provides a set of smooth corner radius values defined by the `radii` key.